test/function_test.go

```diff
@@ -59,13 +59,13 @@ func TestList(t *testing.T) {
 		expectedBody:       url,
 	}
 
-	var data []map[string]map[string]interface{}
+	var data []map[string]interface{}
 	resp, err := http.Get(test.url)
 	assert.NoError(err)
 	actual, err := ioutil.ReadAll(resp.Body)
 	assert.NoError(err)
 	defer resp.Body.Close()
 	json.Unmarshal([]byte(actual), &data)
 	assert.Equal(test.expectedStatusCode, resp.StatusCode, test.description)
-	assert.Equal(test.expectedBody, data[0]["Value"]["srcUrl"], test.description)
+	assert.Equal(test.expectedBody, data[0]["srcUrl"], test.description)
```

Comment:
Test related to `/list` endpoint. Will add more tests related to pull based model.